tests/assert_ansible_schema.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+
+import argparse as ap
+import re
+import sys
+
+import yaml
+
+
+COMMENT_KEYS = set((
```

Comment:
superfluous parentheses

Reply:
Not really, `set` accepts iterable as it's only argument. So it is a set defined using a tuple.

Reply:
I'm sorry, I meant that you should define the set normally.

Reply:
Clarification: Writing `my_set = {1, 2, 3}` is the best way to define a set, but it is not available on python<2.7, and we still have python2.6 in our CI.